demo/app/lib/views/custom_error_expanded.dart

```diff
@@ -0,0 +1,111 @@
+import 'package:flutter/material.dart';
+import 'package:expansion_tile_card/expansion_tile_card.dart';
+
+class ExpansionTileCardDemo extends StatefulWidget {
+
+  @override
+  _ExpansionTileCardDemoState createState() => _ExpansionTileCardDemoState();
+}
+
+class _ExpansionTileCardDemoState extends State<ExpansionTileCardDemo> {
+  final GlobalKey<ExpansionTileCardState> cardA = new GlobalKey();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        automaticallyImplyLeading: false,
+        title: Text('ExpansionTileCard Demo'),
+      ),
+      body: ListView(
+        children: <Widget>[
+          Padding(
+            padding: const EdgeInsets.symmetric(horizontal: 20.0,vertical: 20),
+            child: ExpansionTileCard(
+              baseColor: Colors.cyan[50],
+              expandedColor: Colors.red[50],
+              key: cardA,
+              leading: CircleAvatar(
+                  child: Image.asset("assets/images/devs.jpg")),
+              title: Text("Flutter Dev's"),
+              subtitle: Text("FLUTTER DEVELOPMENT COMPANY"),
```

Comment:
?

Reply:
Some placeholder text?

Reply:
Please refresh the patch this file was for testing some widget not required